tsconfig.json

```diff
@@ -3,8 +3,8 @@
     "declaration": true,
     "declarationDir": "lib",
     "module": "esnext",
-    "target": "es5",
-    "lib": ["es5", "es6", "es7", "es2017", "dom"],
+    "target": "ES2015",
+    "lib": ["ES2015", "ES2016", "ES2017", "DOM"],
```

Comment:
* Upgrade `es5` to `es6`
* Remove `es5` in `lib`
* Replace `es*` with `ES20**` to keep  the style consistent with JS SDK.